corehq/ex-submodules/pillowtop/tests/test_elasticsearch.py

```diff
@@ -195,6 +205,17 @@ def test_replace_doc(self):
         del doc['property']
         self._send_to_es_and_check(doc, update=True)
 
+    def test_merge_doc(self):
+        doc = {'_id': uuid.uuid4().hex, 'doc_type': 'MyCoolDoc', 'property': 'bar'}
+        self._send_to_es_and_check(doc)
+
+        update = doc.copy()
+        del update['property']
+        update['new_prop'] = 'new_val'
+        self._send_to_es_and_check(update, update=True)
+        # merging should still keep old 'property'
+        self._send_to_es_and_check(update, update=True, es_merge_update=True)
```

Comment:
I have updated[ `_send_to_es_and_check` ](https://github.com/dimagi/commcare-hq/pull/15441/files#diff-56cd81bc88100a7354b093f20e6da7c3R184) to reflect above, doesn't look right?